project2/style.css

```
@@ -0,0 +1,232 @@
+    border-radius: 50%;
+    box-shadow: 0 0 0 4px hsla(0, 0%, 0%, 0);
+}
+
+/* screw these old buttons get them out of here */
+[type="radio"] {
+    border: 0;
+    clip: rect(0 0 0 0);
+    height: 1px;
+    margin: -1px;
+    overflow: hidden;
+    padding: 0;
+    position: absolute;
+    width: 1px;
+}
+
+/* custom styling for BEFORE and AFTER active states */
+[type="radio"]+span::before {
+    content: '';
+    display: inline-block;
+    width: 1em;
+    height: 1em;
+    vertical-align: -0.25em;
+    border-radius: 1em;
+    border: 0.125em solid #fff;
+    box-shadow: 0 0 0 0.15em #000;
+    margin-right: 0.75em;
+    transition: 0.5s ease all;
+}
+
+[type="radio"]:checked+span::before {
+    box-shadow: 0 0 0 0.25em #000;
+}
+
+/* ACTUAL SWATCH COLORS FOR CLOTHES SELECTION */
+input[type="radio"]#mustardbx+span::before {
+    background: hsl(41, 100%, 50%);
+}
+
+input[type="radio"]#forestgreen+span::before {
+    background: hsl(120, 60%, 18%);
+}
+
+input[type="radio"]#navyblue+span::before {
+    background: hsl(225, 46%, 19%);
+}
+
+input[type="radio"]#rustred+span::before {
+    background: hsl(4, 81%, 29%);
+}
+
+
+
+/* back to text elements for descriptory info */
+.price {
+    font-weight: 600;
+    margin: 0;
+    padding-top: 15px;
+}
+
+h2 {
+    font-weight: 400;
+    margin: 0;
+    padding: 0;
+    font-size: inherit;
+}
+
+
+a:link,
+a:visited {
+    color: hsl(4, 73%, 62%);
+    display: inline-block;
+    padding: 0 0 0 5px;
+    margin: 0;
+}
+
+.reviewlink:hover {
+    color: hsl(177, 48%, 29%);
+    text-decoration: underline;
+}
+
+
+/* ADD TO CART BUTTON ANIMATION */
+.hvr-sweep-to-top {
+    box-shadow: 0 0 1px rgba(0, 0, 0, 0);
+    display: inline-block;
+    position: relative;
+    transform: perspective(1px) translateZ(0);
+    transition-duration: 0.3s;
+    transition-property: color;
+    -webkit-transform: perspective(1px) translateZ(0);
+    -webkit-transition-property: color;
+    -webkit-transition-duration: 0.3s;
+    vertical-align: middle;
+}
+
+.hvr-sweep-to-top:before {
+    background: hsl(177, 48%, 29%);
+    bottom: 0;
+    content: "";
+    left: 0;
+    position: absolute;
+    right: 0;
+    top: 0;
+    transition-duration: 0.3s;
+    transition-property: transform;
+    transition-timing-function: ease-out;
+    transform: scaleY(.3);
+    transform-origin: 50% 100%;
+    -webkit-transition-property: transform;
+    -webkit-transition-duration: 0.3s;
+    -webkit-transition-timing-function: ease-out;
+    -webkit-transform: scaleY(.3);
+    -webkit-transform-origin: 50% 100%;
+}
+
+.hvr-sweep-to-top:hover,
+.hvr-sweep-to-top:focus,
+.hvr-sweep-to-top:active {
+    color: hsl(0, 0%, 100%);
+}
+
+.hvr-sweep-to-top:hover:before,
+.hvr-sweep-to-top:focus:before,
+.hvr-sweep-to-top:active:before {
+    -webkit-transform: scaleY(1);
+    transform: scaleY(1);
+}
+
+
+/* TESTING REVIEW STAR RATINGS */
+.star-rating {
+    direction: rtl;
+    display: inline-block;
+    padding: 0px;
```

No units are needed on zero measurements.
```suggestion
    padding: 0;
```